vagrant-pxe-harvester/ansible/roles/harvester/templates/config-create.yaml.j2

```diff
@@ -10,9 +10,14 @@ os:
   password: {{ settings['harvester_config']['password'] }}
 install:
   mode: create
-  mgmt_interface: {{ settings['harvester_network_config']['cluster'][0]['mgmt_interface'] }}  # The management interface name
   networks:
-    - interface: {{ settings['harvester_network_config']['cluster'][0]['vagrant_interface'] }}
+    {{ settings['harvester_network_config']['cluster'][0]['vagrant_interface'] }}:
```

Comment:
Is the change superseded by https://github.com/harvester/ipxe-examples/pull/23?

Reply:
Yes. Thanks.